### ISSUE TITLE
[FEATURE] UEBA | Overview page

### ISSUE THREAD

**Issue by jovancacvetkovic** [opensearch-project/security-analytics-dashboards-plugin#407]
**Is your feature request related to a problem?**
Create UEBA overview page.

**What solution would you like?**
The page should have inference visualization data and table data, also there should be recently created aggregators and inference model lists.

**Do you have any additional context?**
![ueba overview page](https://user-images.githubusercontent.com/28289071/218704429-4964beaf-c667-4f43-bcca-d7ccaf4e9fad.png)

**Comment by jovancacvetkovic** [comment]
> 

@getsaurabh02 Yes, I am actively working on this. And will be adding new issues with new mocks.

**Comment by yoicomment** [comment]
Interesting feature. Are you still working on it?
